blint/binary.py

```diff
@@ -907,7 +907,7 @@ def add_elf_dynamic_entries(dynamic_entries, metadata):
     if isinstance(dynamic_entries, lief.lief_errors):
         return metadata
     for entry in dynamic_entries:
-        if entry.tag == lief.ELF.DynamicEntry.TAG.NULL:
+        if not hasattr(entry, 'name'):
```

Comment:
If the type is `DynamicEntryRunPath`, we need to capture the `runpath` attribute.

https://lief.re/doc/stable/doxygen/classLIEF_1_1ELF_1_1DynamicEntryRunPath.html#acea1d603d89cc0ccb0c2284a9f9f595e

Reply:
Added suggested changes, please suggest